Style corrections for 5 chapter of the book

grigory-rechistov opened the pull request.

radare commented:
Not sure if properties is the correct word here. Maybe characteristics or so?



> On 09 Aug 2015, at 16:02, Grigory Rechistov <notifications@github.com> wrote:
> 
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/radare/radare2book/pull/5
> 
> Commit Summary
> 
> rabin2: fixed style, corrected descriptions of options.
> radiff2: beautified text.
> rahash2: corrected style etc.
> rasm2: fixed style, actualized it (e.g., SYNTAX environment var does not work anymore)
> refcard: minor edit of style
> File Changes
> 
> M rabin2/entrypoints.md (4)
> M rabin2/file_identification.md (6)
> M rabin2/imports.md (2)
> M rabin2/intro.md (7)
> M rabin2/libraries.md (6)
> M rabin2/program_sections.md (6)
> M rabin2/strings.md (6)
> M rabin2/symbols.md (6)
> M radiff2/intro.md (2)
> M radiff2/radiff_tool.md (43)
> M rahash2/intro.md (10)
> M rahash2/rahash_tool.md (10)
> M rasm2/assemble.md (12)
> M rasm2/disassemble.md (4)
> M rasm2/intro.md (10)
> M refcard/intro.md (2)
> Patch Links:
> 
> https://github.com/radare/radare2book/pull/5.patch
> https://github.com/radare/radare2book/pull/5.diff
> —
> Reply to this email directly or view it on GitHub.
>

grigory-rechistov commented:
> Not sure if properties is the correct word here. Maybe characteristics or
so?
Do you mean the word in "Rabin2 — Show *Properties* of a Binary" title? I
didn't understand for sure what line your comment is about.
If so, then maybe "Rabin2 — Show Characteristics of a Binary" will be
better?

2015-08-09 21:01 GMT+03:00 radare <notifications@github.com>:

> Not sure if properties is the correct word here. Maybe characteristics or
> so?
>
>
>
> > On 09 Aug 2015, at 16:02, Grigory Rechistov <notifications@github.com>
> wrote:
> >
> >
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/radare/radare2book/pull/5
> >
> > Commit Summary
> >
> > rabin2: fixed style, corrected descriptions of options.
> > radiff2: beautified text.
> > rahash2: corrected style etc.
> > rasm2: fixed style, actualized it (e.g., SYNTAX environment var does not
> work anymore)
> > refcard: minor edit of style
> > File Changes
> >
> > M rabin2/entrypoints.md (4)
> > M rabin2/file_identification.md (6)
> > M rabin2/imports.md (2)
> > M rabin2/intro.md (7)
> > M rabin2/libraries.md (6)
> > M rabin2/program_sections.md (6)
> > M rabin2/strings.md (6)
> > M rabin2/symbols.md (6)
> > M radiff2/intro.md (2)
> > M radiff2/radiff_tool.md (43)
> > M rahash2/intro.md (10)
> > M rahash2/rahash_tool.md (10)
> > M rasm2/assemble.md (12)
> > M rasm2/disassemble.md (4)
> > M rasm2/intro.md (10)
> > M refcard/intro.md (2)
> > Patch Links:
> >
> > https://github.com/radare/radare2book/pull/5.patch
> > https://github.com/radare/radare2book/pull/5.diff
> > —
> > Reply to this email directly or view it on GitHub.
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/radare/radare2book/pull/5#issuecomment-129222334>.
>



-- 
Best regards,
Grigory Rechistov
